nginx/assets/dashboards/plus_overview.json

```diff
@@ -3,934 +3,1269 @@
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "response_format": "timeseries",
+                  "queries": [
+                    {
+                      "query": "avg:nginx.ssl.handshakes_failed_count{*}.as_count()",
+                      "data_source": "metrics",
+                      "name": "query1"
+                    }
+                  ],
+                  "style": {
+                    "palette": "dog_classic",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "bars"
+                }
+              ],
+              "yaxis": {
+                "scale": "linear",
+                "label": "",
+                "include_zero": true,
+                "min": "auto",
+                "max": "auto"
+              },
+              "markers": []
+            },
+            "layout": {
+              "x": 0,
+              "y": 4,
+              "width": 6,
+              "height": 2
+            }
           }
         ],
-        "custom_links": [],
-        "title": "Dropped connections",
-        "title_size": "16",
-        "title_align": "left",
-        "precision": 2
+        "background_color": "vivid_green",
+        "isShared": false,
+        "subtype": "group"
       },
       "layout": {
-        "x": 66,
-        "y": 17,
-        "width": 21,
-        "height": 10
-      }
-    },
-    {
-      "id": 4824885675186612,
-      "definition": {
-        "type": "note",
-        "content": "Sampling your request data (**requests** in open source, or **total** in Plus) with a fixed time interval provides you with the number of requests you’re receiving per unit of time—often minutes or seconds. \n\nMonitoring this metric can alert you to spikes in incoming web traffic, whether legitimate or nefarious, or sudden drops, which are usually indicative of problems. A drastic change in requests per second can alert you to problems brewing somewhere in your environment, even if it cannot tell you exactly where those problems lie. Note that all requests are counted the same, regardless of their URLs.\n\n",
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "left"
-      },
-      "layout": {
-        "x": 165,
+        "x": 6,
         "y": 0,
-        "width": 28,
-        "height": 31
+        "width": 6,
+        "height": 7
       }
     }
   ],
-  "template_variables": [
-    {
-      "name": "scope",
-      "default": "*",
-      "prefix": "host"
-    }
-  ],
-  "layout_type": "free",
+  "template_variables": [],
+  "layout_type": "ordered",
   "is_read_only": false,
```

Comment:
Should we set this to true maybe ? or the fact that it's an integration dashboard makes it read only by default ? 🤔

Reply:
It shouldn't make the dashboard editable anyway but I agree it's more straightforward to have it set to true